app/app/utils.tsx

```diff
@@ -475,6 +475,7 @@ export interface MemberStats {
 }
 
 export const DECIMALS = 1e8;
+export const DECIMAL_PLACES = 8;
```

Comment:
I think we should specify that this decimal place value is specific to btc, bch, and doge.

Reply:
Good catch, using pool.nativeAssetDecimal instead.